gc/base/GCExtensionsBase.hpp

```diff
@@ -744,6 +744,10 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 	bool scavengerAlignHotFields; /**< True if the scavenger is to check the hot field description for an object in order to better cache align it when tenuring (enabled with the -Xgc:hotAlignment option) */
 	uintptr_t suballocatorInitialSize; /**< the initial chunk size in bytes for the J9Heap suballocator (enabled with the -Xgc:suballocatorInitialSize option) */
 	uintptr_t suballocatorCommitSize; /**< the commit size in bytes for the J9Heap suballocator (enabled with the -Xgc:suballocatorCommitSize option) */
+	uintptr_t suballocatorIncrementSize;
+#if defined(OMR_ENV_DATA64) && defined(LINUX)
+	bool suballocatorQuickAlloc;
+#endif /* defined(OMR_ENV_DATA64) && defined(LINUX) */
```

Comment:
We should add a description for the new fields similar to the existing ones.

Reply:
Hmm, I had these at one point, not sure what happened to them. Thanks for pointing it out.